docs/linking/2-request-consent-otp.puml

```diff
@@ -56,6 +56,12 @@ Switch -> DFSP ++:  ""POST /consentRequests""\n\
 DFSP --> Switch: ""202 Accepted""
 deactivate Switch
 
+DFSP -> DFSP : ""POST /sendOTP""\n\
```

Comment:
DFSP to DFSP? just double checking that this is correct.

Reply:
I didn't want to introduce a new actor. I want to only illustrate that DFSP is responsible to send SMS. This endpoint will be needed when we will implement linking flow in `thridparty-scheme-adapter`.

Reply:
👍